### ISSUE TITLE
iOS 17.0.1 not starting the service com.apple.testmanagerd.lockdown.secure

### ISSUE THREAD

**Issue by gusoard** [danielpaulus/go-ios#285]
**Describe the bug**
Is not possible to start the web driver agent server due to this error:

`service:com.apple.testmanagerd.lockdown.secure with reason:'InvalidService'`

This issue started on the test iPhone once updated to iOS 17.0.1. Before, it was working like a charm. I add information of another iPhone with iOS 16.3. 

Both iPhones have the same WDA app installed, compiled from the same WDA project (latest appium xcuidriver and with appium 2)

Is weird, since when launching the test from xCode, it works OK (the server is started)

**To Reproduce**
1. Install the WDA app in an iPhone with iOS 17.0.1
2. Install latest go-ios (v1.0.116 to this date)
3. Execute "ios runwda" on the iPhone

**Expected behavior**
WDA started using iOS 17.0.1

**Screenshots**
go-ios version
`➜  ~ ios --version
{"version":"v1.0.116"}`

Devices list:
<img width="1751" alt="image (1)" src="https://github.com/danielpaulus/go-ios/assets/5107662/c6ed69e1-7ea2-45d0-ad90-3d856b4fe92f">

Failing iOS 17.0.1

<img width="1703" alt="image" src="https://github.com/danielpaulus/go-ios/assets/5107662/434636ba-764f-4019-b862-f43343b9040a">

Working iOS 16.3

<img width="849" alt="image (2)" src="https://github.com/danielpaulus/go-ios/assets/5107662/d0c4918f-7829-4049-9040-5bf3346f1655">

**Desktop (please complete the following information):**
 - OS: macOS Ventura
 - Version: 13.6

**Smartphone (please complete the following information):**
 - Device: iPhone 14
 - OS: iOS 17.0.1

**Additional context**
Same iPhone, before updating to iOS 17.0.1 started the server without issues

**Comment by shamanec** [comment]
Yes, currently `go-ios` as most similar tools does not support the majority of functionality on iOS 17 due to changes in the communication from Apple side, as well as changes on DDI. I believe `pymobiledevice3` already has some progress on this but only on macOS if I am not mistaken. Getting it to work on other platforms as well is probably gonna be a major effort but our amazing @danielpaulus has got this! ❤️

**Comment by shamanec** [comment]
> Awesome! Thanks for answering so quickly! I've been checking pymobiledevice3 during the day, but still having some issues (on Mac), plus not understanding much of it tbh hahaha.
> 
> In the meanwhile I'm launching WDA from xcode manually and after launching appium to make it work 😅

Yeah, I do the same for iOS 17, its a bit of an unfortunate turn of events :D Fingers crossed that can change :)

**Comment by danielpaulus** [comment]
Closing during issue triage: this report is tied to an old go-ios/iOS version and we do not have a current reproduction. Please retest with the latest go-ios release and open a new issue, or ask us to reopen this one, with fresh logs if it still reproduces.
